docs/pages/docs/faq.mdx

```diff
@@ -52,6 +52,174 @@ defined and incompatible interfaces - which are the biggest problems in common
+5. Quint has the concept of a `run` to define executions to be used as tests.
+   There is no corresponding feature in TLA+.
+6. TLA+ has model values and symmetry sets, which are not available in Quint at
+   this time.
+ 
+On a less concrete note, TLA+ supporters will often say that the mathematical
+syntax helps to put people in the correct mindset of formal specification,
+while Quint as a project hopes to teach people this mindset without a big syntax
+and tool disruption.
+</FAQBox>
+
+<FAQBox title="Is Quint the same thing as PlusCal?">
+No. Although both Quint and PlusCal have the goal of making TLA+ more
+approachable to engineers, the strategies are very different. PlusCal uses the
+same tooling and similar syntax to TLA+, but introduces imperative constructs
+(while TLA+ is fully declarative). Quint, on the other hand, introduces new
+tooling and syntax, but keeps the declarative aspect.
+
+It would be possible to add the imperative constructs from PlusCal to Quint. We
+currently don't have a lot of clarity on how useful that would be.
+
+We experienced situations where the expressiveness constraints imposed by PlusCal were a
+problem for some of our systems, while Quint has always been enough.
+</FAQBox>
+
+
+<FAQBox title="How does Quint compare to Alloy?">
+
+Alloy is very focused on sets, and is good for specifying requirements and data
+structures, while Quint is natively time oriented, fitting really well with
+concurrent and distributed systems. As everything in Alloy is a set/relation, it
+requires a quite abstract and specific way of modeling. 
+
+While Alloy 6 added temporal operators, for the longest time,
+handling the passage of time required complex workarounds, and
+therefore having the language be used mostly for other things.
+
+Alloy verification can be faster than Quint for specifications without time, as
+it can use SAT solving instead of SMT solving, and the tooling looks very
+different, with Alloy Analyzer being very visual.
+
+</FAQBox>
+
+<FAQBox title="How does Quint compare to Coq/Isabelle/Lean?">
+
+Coq, Isabelle and Lean are proof assistants. They will assist you to write a
+proof, but you still have to do most of the work. Quint uses model checking,
+where you only need to define the model and the properties, and the verification
+process is fully automated. Learning and mastering how to use a proof assistant
+is much harder (usually taking years) than learning and mastering Quint.
+
+Proof assistants require you to write deductive proofs, while model checkers (as
+in Quint), use an inductive approach, exploring all reachable states in a model.
+Proof assistants are mostly used for research, and model checking techniques are
+more present in the industry.
+
+Proving something using one of these tools provides a much stronger guarantee
+than verifying something with a model checker. Proofs derive directly from
+constructive mathematics, while model checkers are complex implementations on
```

Comment:
Oh, my bad! I thought all of them were used constructive logics. Do you have any suggestions on what would be more accurate here? I just want to say that there is a really low chance that there is an implementation error on the proof assistants that would result in wrong proofs being checked, while there is much more room for mistakes in model checker implementations.

Reply:
Proof assistants are software and thus can have bugs too. They are generally more trustworthy than model checkers because these proof assistants in particular rely on a small kernel implementing the core logical rules. All proofs reduce to the rules in the kernel. The kernel is very small, probably a few hundred lines of code, so it's easier to check and trust that it faithfully implement the rules.

Reply:
That's good, thanks!